components/feature/prompts/src/main/java/mozilla/components/feature/prompts/PromptFeature.kt

```diff
@@ -424,6 +425,7 @@ class PromptFeature private constructor(
                 is File -> filePicker.handleFileRequest(promptRequest)
                 is Share -> handleShareRequest(promptRequest, session)
                 is SelectCreditCard -> {
+                    emitSuccessfulCreditCardAutofillFormDetectedFact()
```

Comment:
Should this be inside of if below line 429?

Reply:
No, we still want to emit the fact even if credit card autofill is disabled or there is not credit card stored.  The reason is the fact is just to show that we've detected a credit card autofill form.